app/assets/javascripts/application.js

```diff
@@ -54,4 +54,8 @@ $(document).ready(function(){
       }
     });
   });
+
+  $(function(){
+    $(':input[type=number]').on('mousewheel',function(e){ $(this).blur(); });
+  });
```

Comment:
Cool. Seems like you opted for a JS solution to keep the field type as number. For that let's make use of the number field and as a `min` option for it (minimum team size should be 1 and min tech team size should be 0)